### ISSUE TITLE
Windows - Step 4 - Raw Mode

### ISSUE THREAD

**Issue by mattmccarty** [ErichDonGubler/termion#5]
Is this something that Windows supports? I'm not sure. Research is needed.

**Comment by ErichDonGubler** [comment]
Yes, there's an equivalent of raw mode already implemented. It's not perfect, but it's been doable so far. My biggest doubt with this is how it's going to work with alternative terminal emulators like Cmder, ConsoleZ, etc.
